cnd/src/config/file.rs

```diff
@@ -102,7 +102,6 @@ pub struct Bitcoin {
 
 #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
 pub struct Ethereum {
-    pub network: ethereum_support::Network,
```

Comment:
I think last time we touched this, we ought to keep it because it is the primary way we want to specify things, even though we are not using yet :/

Reply:
See here https://github.com/comit-network/comit-rs/issues/1455.

Reply:
If it is not use that not only it's useless but also dangerous are we luring the user in thinking it's used and possibly that some checks are done.

To be re-introduced with #1455 then.